packages/editor/src/components/block-switcher/index.js

```diff
@@ -61,15 +61,11 @@ export class BlockSwitcher extends Component {
 				return null;
 			}
 			return (
-				<Toolbar>
-					<IconButton
-						disabled
-						className="editor-block-switcher__no-switcher-icon"
-						label={ __( 'Block icon' ) }
-					>
+				<div className="components-block-switcher" aria-hidden="true">
```

Comment:
I don't think, we should be re-creating `Toolbar` here as this is still the same `div`.